eea/facetednavigation/docs/unicode.txt

```diff
@@ -37,7 +37,7 @@ Add some content
     >>> doc = sandbox.invokeFactory('News Item', 'non-ascii-folder', title='NON-ASCII Folder')
     >>> doc = sandbox._getOb(doc)
     >>> doc.setLanguage('en')
-    >>> doc.setSubject((u'\u015etire',))
+    >>> doc.setSubject(('$tire',))
```

Comment:
:) That should actually be `Ș`. Would be also ok to replace it with any non-ascii french char

Reply:
Sure, I replaced it with "é", moreover $ is an ascii char so the test was completely missing it's target...

Jenkins should be running again...